stdlib/ldap/src/main/java/org/ballerinalang/stdlib/ldap/util/LdapUtils.java

```diff
@@ -36,6 +36,8 @@
 import javax.naming.directory.SearchControls;
 import javax.naming.directory.SearchResult;
 
+import static org.ballerinalang.stdlib.ldap.LdapConstants.LDAP_PACKAGE_ID;
```

Comment:
Can we remove the static import?

Reply:
These static imports do not damage the code and it improves the readability. Hence resolving